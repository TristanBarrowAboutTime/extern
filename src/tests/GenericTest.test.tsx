import * as React from 'react';
import * as renderer from 'react-test-renderer';

describe('Edit Folder Row', () => {
    it('renders Correctly', () => {
        const tree = renderer.create(
            <div>Generic Test</div>
        ).toJSON();   
        expect(tree).toMatchSnapshot();    
    });
});