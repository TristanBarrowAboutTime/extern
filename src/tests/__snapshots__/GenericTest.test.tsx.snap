// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Edit Folder Row renders Correctly 1`] = `
<div>
  Generic Test
</div>
`;
